cardano-node/src/Cardano/Node/ToObjectOrphans.hs

```diff
@@ -0,0 +1,181 @@
+{-# LANGUAGE FlexibleContexts      #-}
+{-# LANGUAGE FlexibleInstances     #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE ScopedTypeVariables   #-}
+{-# LANGUAGE UndecidableInstances  #-}
+{-# OPTIONS_GHC -fno-warn-orphans  #-}
+
+module Cardano.Node.ToObjectOrphans
+  (
+    WithTip (..)
+  , showTip
+  , showWithTip
+  ) where
+
+import           Cardano.Prelude hiding (atomically, show)
+import           Prelude (String, show, id)
+
+import           Data.Aeson (Value (..), toJSON, (.=))
+import           Data.Text (Text, pack)
+import           Cardano.BM.Data.Tracer (ToObject (..),
+                                         TracingVerbosity (..),
+                                         Transformable (..),
+                                         emptyObject, mkObject, trStructured)
+
+import           Ouroboros.Consensus.Ledger.Abstract
+import           Ouroboros.Consensus.Util.Condense
+import qualified Ouroboros.Network.AnchoredFragment as AF
+import           Ouroboros.Network.Block
+import qualified Ouroboros.Storage.ChainDB as ChainDB
+import qualified Ouroboros.Storage.LedgerDB.OnDisk as LedgerDB
+
+
+-- | Tracing wrapper which includes current tip in the logs (thus it requires
+-- it from the context).
+--
+-- TODO: this should be moved to `ouroboros-consensus`.  Running in a seprate
+-- STM transaction we risk reporting  wrong tip.
+--
+data WithTip blk a =
+    WithTip
+      (Point blk)
+      -- ^ current tip point
+      a
+      -- ^ data
+
+showWithTip :: Condense (HeaderHash blk)
```

Comment:
Ah, if you don't want the log output to change when we change our `Condense` instance for `HeaderHash blk` (which, however unlikely, might happen), then you should roll your own version of it (`pprintPoint`?)

Reply:
removed the `Condense` constraint
what do you think?